### ISSUE TITLE
Remove comments in <style> block

### ISSUE THREAD

**Issue by jkupczak** [codsen/email-remove-unused-css#15]
Is it within the scope of this project to handle the removal of comments in the CSS block (and perhaps HTML comments)? 

Perhaps as an extra fix on https://emailcomb.com/ along with the HEX shorthand and image alt attributes?

**Comment by revelt** [comment]
Yeah, it's doable. We are traversing head styles anyway, it's just a matter of catching where comments start and end. I'll put onto the to-do list.

Sorry the front-ends are not using up-to-date API, when I updated create-react-app it broke and I need to migrate from stale codemirror to new-one, which warrants a full recode pretty much...
